frontend/src/index.pug

```diff
@@ -272,21 +275,15 @@ html
                 )
                   i.button.fas.fa-user-edit
               pre.hljs
-                div(v-hljs="file.path")
-                  template(v-for="segment in file.segments")
-                    .segment(v-bind:class="{ untouched: !segment.authored, active: segment.lines.length<5 }")
-                      .closer(v-if="!segment.authored && segment.lines.length>4", onclick="toggleNext(this)")
-                        i.fas.fa-ellipsis-h.icon
-                        span.tooltiptext.tooltiptext--close Show untouched code
-                        span.tooltiptext.tooltiptext--open Hide untouched code
-                      .code(v-for="(line, index) in segment.lines")
-                        .line-number {{ segment.lineNumbers[index] + "\n" }}
-                        .line-content {{ line + "\n" }}
+                template(v-for="segment in file.segments")
```

Comment:
I think we don't need a `template` here, as `v_segment' itself is a template. A div is enough.